app/src/main/java/com/cmput301f20t21/bookfriends/services/BookService.java

```diff
@@ -0,0 +1,31 @@
+package com.cmput301f20t21.bookfriends.services;
+
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.QuerySnapshot;
+
+public class BookService {
+    // TODO not done
+
+    private CollectionReference bookCollection;
+
+    private static final BookService instance = new BookService();
+
+    private BookService() {
+        bookCollection = FirebaseFirestore.getInstance().collection("books");
+    }
+
+    public static BookService getInstance() {
+        return instance;
+    }
+
+    public Task<QuerySnapshot> getBooksOfOwnerID(String uid) {
```

Comment:
This can be combine with the one below to be getBooksByUID?

Reply:
how do we figure out the books that's borrowed by this user?
Option 1: have a borrower field in Book
Option 2: query for the user's requests whose book is borrowed.

Reply:
I think option 2 is better

Reply:
I prefer option 2 as well

Reply:
We will need the Request entity for that functionality. In another PR, maybe?